### PR TITLE
Update tokens_optimism_erc20.sql

### PR DESCRIPTION
**Pull request by 0xToli**
Added Reaper.Farm Vault tokens.

Brief comments on the purpose of your changes:

*For Dune Engine V2*
I've checked that:
General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributors)

Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

File: models/tokens/optimism/reaper_all_vaults.sql [hunk 1]
```diff
@@ -0,0 +1,66 @@
+{{ config( alias='erc20')}}
+SELECT LOWER(contract_address) AS contract_address, symbol, decimals
+  FROM (VALUES 
+
+,('0x39FdE572a18448F8139b7788099F0a0740f51205','OATH', 18)
```
Comment (Hosuke) [inline models/tokens/optimism/reaper_all_vaults.sql:5]: Remove the extra comma here then it should be able to pass the dbt run

Comment (jeff-dude) [reply models/tokens/optimism/reaper_all_vaults.sql:5]: @0xToli any reason this view needs to exist if you add the same tokens to the existing erc20 optimism view below?

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: models/tokens/optimism/reaper_all_vaults.sql (hunk 1 above)
Comment (jeff-dude) [reply models/tokens/optimism/reaper_all_vaults.sql:5]: please note merge of #1783 -- i believe that makes this add here irrelevant

**Comment by jeff-dude** [comment]
closing due to inactivity, please do submit new PR when ready

**Pull request closed without merging**